src/main/java/com/powsybl/openloadflow/network/LfAction.java

```diff
@@ -193,13 +193,18 @@ public void updateConnectivity(GraphConnectivity<LfBus, LfBranch> connectivity)
         }
     }
 
-    public void apply() {
+    public void apply(LfNetwork network, boolean dc) {
         if (tapPositionChange != null) {
             LfBranch branch = tapPositionChange.getBranch();
             int tapPosition = branch.getPiModel().getTapPosition();
             int value = tapPositionChange.getValue();
             int newTapPosition = tapPositionChange.isRelative() ? tapPosition + value : value;
             branch.getPiModel().setTapPosition(newTapPosition);
+            if (dc) { // FIXME
```

Comment:
@geofjamg I think that it is not the good fix, maybe it will be better in the setTapPosition but I prefer to ask for our opinion before. We need to update the target vector because right hand side has changed.

Reply:
Indeed right place is in setTapPosition.
Done.